src/_P035_IRTX.ino

```diff
@@ -235,6 +241,8 @@ boolean Plugin_035(byte function, struct EventStruct *event, String& string)
             }
 
             Plugin_035_irSender->sendRaw(buf, idx, IrHz);
+            delete buf;
```

Comment:
When allocating memory using `new bla[...]` (thus using the `new[]` operator) you should use the `delete[]` operator.
It is the 'array' version of just a single element.